lib/ansible/plugins/strategy/__init__.py

```diff
@@ -318,6 +318,14 @@ def _queue_task(self, host, task, task_vars, play_context):
                 self._cur_worker += 1
                 if self._cur_worker >= len(self._workers):
                     self._cur_worker = 0
+                if len(task.forks) > 0 and task._parent._play.strategy != "free":
+                    if task.run_once:
+                        display.warning("Ignoring 'forks' as 'run_once' is also set for '%s'" % task.get_name())
```

Comment:
i dont think this requires a warning, since min forks is 1 and run_once will be the 'min'

Reply:
I can remove this again, but you have been asking for this before: "Another example, when combining max_concurrent: X with run_once ... i would also expect a warning or error since those settings are in conflict and we can only obey one. Its simpler with serial/fork since run_once is clearly more specific"

Reply:
same as run_once and serial and --forks, it takes precedence, as long as that is clearly documented  about the keywords i dont see need for runtime warning

Reply:
Agreed.

Reply:
O.k, these messages are now a debug messages.